src/ventus/wm/window.js

```diff
@@ -486,8 +489,23 @@ function(Emitter, Promise, View, WindowTemplate) {
 		},
 
 		move: function(x, y) {
-			this.x = x;
-			this.y = y;
+			var maxX = window.innerWidth - this._margin.right;
+			var maxY = window.innerHeight - this._margin.bottom;
```

Comment:
Yes but there are cases were Ventus can be inserted in a element using only half of the page while in the other half other component exists like a half-desktop half-console (someone said [Terminus](https://github.com/rlamana/Terminus) :P) application. In that case I guess it's not right to consider the other component a "margin" but Ventus should limit itself to it's container. Please correct me if I'm wrong.